frontend/src/container/CreateAlertRule/defaults.ts

```diff
@@ -6,6 +6,16 @@ import {
 	defaultMatchType,
 } from 'types/api/alerts/def';
 
+const defaultAlertDescription =
+	'This alert is fired when the defined metric (current value: {{$value}}) exceeds the threshold ({{$threshold}})';
```

Comment:
Wouldn't this be incorrect when they changed the criteria _metric is below x ..._/_metric is equal to x..._?

Reply:
This is more like a suggested way to write description. The default alert is above condition. User can change it when they go for below condition.

Reply:
If I were the user, I would probably forget to update the description. It would be good if the default description changes with the condition so there isn't another user input action needed.

Reply:
@mindhash @srikanthccv We can reword as - 

current value: {{$value}}) *crosses* the threshold 

crosses can mean both exceed and go below

Reply:
Done